command/repo.go

```diff
@@ -179,6 +179,23 @@ func addUpstreamRemote(cmd *cobra.Command, parentRepo ghrepo.Interface, cloneDir
 func repoCreate(cmd *cobra.Command, args []string) error {
 	projectDir, projectDirErr := git.ToplevelDir()
 
+	if projectDirErr == nil {
+		remoteExist, err := git.CheckRemoteExist()
```

Comment:
I would prefer a function called `isRemoteConfigured` that takes a remote name string and returns a boolean.